PACKAGE = "image_editor"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("roomba0_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba1_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba2_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba3_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba4_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba5_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba6_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba7_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba8_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)
gen.add("roomba9_whitebalance_threshold", double_t, 0, "whitebalance", 0.0, 0.0, 1.0)

exit(gen.generate(PACKAGE, "image_editor", "Parameter"))
